libkbfs/kbfs_cr_test.go

```diff
@@ -24,12 +24,12 @@ func readAndCompareData(t *testing.T, config Config, ctx context.Context,
 	kbfsOps := config.KBFSOps()
 	fileNode, _, err := kbfsOps.Lookup(ctx, rootNode, "a")
 	if err != nil {
-		t.Fatalf("Couldn't lookup file: %v", err)
+		t.Fatalf("Couldn't lookup file: %+v", err)
```

Comment:
The only change in this file is '%v", err' -> '%+v", err', and handling the errors from `EnableJournaling`.